admin-front/src/components/TimeSeries/BarChartTimeSeries.tsx

```diff
@@ -6,14 +6,16 @@ interface BarChartTimeSeriesProps {
   timeSeriesType: TimeSeriesType;
   timeStamps: string[];
   timeStampValues: number[];
+  currencySymbol?: string;
 }
 
 const BarChartTimeSeries = ({
   timeSeriesType,
   timeStamps,
   timeStampValues,
+  currencySymbol,
 }: BarChartTimeSeriesProps) => {
-  const isPriceVolume = timeSeriesType === 'priceVolume';
+  const isPriceVolume = timeSeriesType === 'priceVolume' && !!currencySymbol;
```

Comment:
I don't think the `isPriceVolume` should become `false` if `currencySymbol` is undefined. Rather than we should just have a "Price" label without the currency symbol (as it was before).

Because as it is now it'll show "Count" on the y axis label, when we are displaying the price volume and the currency is unknown.